src/DateRangeInput/DateRangeInput.js

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Box, Input } from '@quarkly/widgets';
+import { useOverrides } from '@quarkly/components';
+import { format, isValid } from 'date-fns';
+import { overrides, propInfo, defaultProps } from './props';
+import DateRangePicker from '../DateRangePicker';
+import { useInput } from '../DateSingleInput/hooks';
+
+const DateRangeInput = ({
```

Comment:
We need to add support for required and name (for minDate and maxDate fields) attributes for inputs